### PR TITLE
feat: query arg propertyies

### PR DESCRIPTION
**Pull request by Laurensvdmaas**

### CODE REVIEW

File: components/globals/Playground.vue [hunk 1]
```diff
@@ -51,6 +61,41 @@ export default class extends Mixins(Base) {
...
 <style lang="scss">
 .playground {
+  --sticky-body-to-down-bg: linear-gradient(
+    180deg,
+    var(--body) 0%,
+    var(--transparent-body) 100%
+  );
+  --sticky-body-to-up-bg: linear-gradient(
+    0deg,
+    var(--body) 0%,
```
[the remark this answers is not in the document]
Comment (Wouter125) [reply components/globals/Playground.vue:71]: We got them available in the package then right? So we could grab them from there?

File: layouts/default.vue [hunk 2]
```diff
@@ -429,6 +429,8 @@ export default class Layout extends Mixins(Base) {
       ].filter((el) => el.id)
       this.stopReplacing = false
     }, 1000)
+
+    console.log(this.$cloudinaryFolder)
```
Comment (Wouter125) [inline layouts/default.vue:433]:
```suggestion
```

File: scripts/schema.js [hunk 3]
```diff
@@ -208,16 +208,15 @@ const appendOfType = (fields, allowRequired = false) => {
...
-    }
-
     const args = appendOfType(
       field?.args || [],
       allowRequired
     );
 
+    if(field.name === 'polyline') {
+      console.log(field)
+    }
```
Comment (Wouter125) [inline scripts/schema.js:218]:
```suggestion
```

Don't need this debuggin anymore.

File: tailwind.config.js [hunk 4]
```diff
@@ -107,6 +107,7 @@ module.exports = {
         80: 0.8,
       },
       fontSize: {
+        0: 0,
```
Comment (Wouter125) [inline tailwind.config.js:110]: Why should we include a font-size of 0?

**Review by Wouter125 — CHANGES_REQUESTED** [review]

File: nuxt.config.js [hunk 5]
```diff
@@ -8,6 +9,7 @@ export default {
     EDIT_GITHUB_URL: process.env.EDIT_GITHUB_URL,
     ALGOLIA_APP_ID: process.env.ALGOLIA_APP_ID,
     ALGOLIA_API_PUBLIC_KEY: process.env.ALGOLIA_API_PUBLIC_KEY,
+    CLUDINARY_FOLDER: process.env.CLUDINARY_FOLDER,
```
Comment (imjasonmiller) [inline nuxt.config.js:12]: Nit: should this be `CLOUDINARY_FOLDER`?

Comment (Wouter125) [reply nuxt.config.js:12]: Yea added it as a suggestion above :)

File: layouts/default.vue (hunk 2 above)
Comment (imjasonmiller) [inline layouts/default.vue:433]: Is this perhaps related to the environment variable from `nuxt.config.js`?

**Review by imjasonmiller — CHANGES_REQUESTED** [review]
It looks great! I've left a couple of comments that might be addressed before merging, but these are nits.

File: components/globals/Playground.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Laurensvdmaas) [reply components/globals/Playground.vue:71]: They go from 80 -> 100 that's why i put them there, removed the up to down ones.

File: nuxt.config.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (Laurensvdmaas) [reply nuxt.config.js:12]: This is not necessary anymore, removed.

File: tailwind.config.js (hunk 4 above)
Comment (Laurensvdmaas) [reply tailwind.config.js:110]: It's in the component library, usually it's used to remove the space between `display: inline-block` items

File: nuxt.config.js (hunk 5 above)
Comment (Laurensvdmaas) [reply nuxt.config.js:12]: Removed this line, not necessary anymore

**Review by Wouter125 — APPROVED** [review]

**Pull request merged**